client/src/components/FormField/RegisteredMFAMethodListField/RegisteredMFAMethodListField.scss

```diff
@@ -0,0 +1,9 @@
+.registered-mfa-method-list-field {
+  padding: .5385rem 0;
```

Comment:
If this is part of the CMS bundle you should be able to import the admin module's variables and use one of those here